build.xml

```diff
@@ -241,7 +241,7 @@
 
 	<target name="phpunit" unless="phpunit.done" depends="phpunit-prepare"
 		description="Run unit tests with PHPUnit">
-		<exec executable="${phpunit}" resultproperty="result.phpunit"
+		<exec executable="${phpunit}" failonerror="true" resultproperty="result.phpunit"
```

Comment:
Just throwing this in there, because that is something that should have been added anyway, because we actually want to fail the entire build when some tests fail, so we know that something is up.

Reply:
This job is supposed to fail too, but since the Tests did not fail, the job will not fail either, even though it really should: https://travis-ci.com/Froxlor/Froxlor/jobs/220526962
I assume that's a bug in the test that deletes a user database, which simply lets these exceptions fly.
https://travis-ci.com/Froxlor/Froxlor/jobs/220526963 (MySQL 5.7) failed, which was expected.
https://travis-ci.com/Froxlor/Froxlor/jobs/220526964 (MySQL 8.0) failed too, nice. That's how it should be.